### PR TITLE
Blender 28 docs

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: docs/about.rst [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+
+    - Project Lead:
+        - Jonathan Dent
 
     - Contributing Authors:
         - François Beaune
+        - João Marcos Mororo Costa
+        - Herbert Crepaz
         - Joel Daniels
         - Jonathan Dent
```
Comment (Mango-3) [inline docs/about.rst]: You put your name twice

File: docs/features.rst [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-    * Adjustable number of motion segments for camera, object, and deformation blur
+    * Blender 2.8 support
+    * Dynamic OSL script node
+    * Texture converter uses internal function (no more need for maketx)
+    * Animated textures
+    * Native Cryptomatte integration
+    * Mesh export is now handled through C++, leading to a 10x reduction in export times
+    * Support for adding and deleting objects in viewport render model
+    * Stereoscopic rendering support
+    * Search paths for shaders can be added in the preferences menu
```
Comment (Mango-3) [inline docs/features.rst]: You probably mean Search paths for custom shader can be easily set.... The default shaders are by default included.

File: docs/installation.rst [hunk 3]
```diff
@@ -3,10 +3,11 @@
...
 ============
 
-Download
-	Download the .zip file of the latest `blenderseed release <https://github.com/appleseedhq/blenderseed/releases>`_ for your platform. appleseed itself is bundled with the addon, so no additional downloads are needed.  Note that as of 0.8.0 you must use Blender 2.79 or higher.
+Version
+    As of version 2.0, blenderseed is intended to be used with Blender 2.8.  We do have a maintenance branch for the Blender 2.79 exporter, but it is not actively developed and will likely be missing features that are present in the main branch.
 
-	*Blender 2.8 is not currently supported.  We will add support for it once the Python API has been finalized.*
+Download
+	Download the .zip file of the latest `blenderseed release <https://github.com/appleseedhq/blenderseed/releases>`_ for your platform and Blender version. appleseed itself is bundled with the addon, so no additional downloads are needed.  Please note that the 2.79 version **requires** Blender 2.79 or later.  It will not work with earlier versions.
```
Comment (Mango-3) [inline docs/installation.rst]: Please note that the 2.79 version **requires** Blender 2.79b Official.

File: docs/installation.rst [hunk 4]
```diff
@@ -22,19 +23,22 @@ Configure
...
 	- APPPLESEED_BIN_DIR:
-		Set to the /bin folder of your appleseed build.
+		Set to the folder of your appleseed build's sandbox/bin folder that contains the main appleseed library.
 
 .. rubric:: Footnotes:
 
 .. [#f1] If you are compiling applessed for use with blenderseed, you will need to compile appleseed with the Python 3 bindings enabled.  Please see the `build instructions <https://github.com/appleseedhq/appleseed/wiki/Building-appleseed>`_.
-.. [#f2] You must also compile with the same version of Python 3 that is used by your Blender install.  Blender 2.79 uses Python 3.5.  Current development snapshots and the 2.8 branch use Python 3.7.
\ No newline at end of file
+.. [#f2] You must also compile with the same version of Python 3 that is used by your Blender install.  Blender 2.79 uses Python 3.5.  2.8 uses Python 3.7.
```
Comment (Mango-3) [inline docs/installation.rst]: Blender 2.79b uses Python 3.5
(During the various Blender 2.79 releases every Pyhon version from 3.4 to 3.7 had been used inside Blender at one point.)

File: docs/panels/object/object_options.rst [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
-- Visibility Flags:
-    These flags determine if an object is visible to specific ray types during rendering.
-- Ray Bias:
-    This is used to offset the starting point of a ray from the surface of an object.
-- Object Alpha:
-    This allows a user to determine the transparency of an object.  The parameter can also be driven by a texture map, allowing for cutout effects like leaves.
 - Double Sided Shading:
     This tells the renderer to apply the object material to both sides of the polygon.  This is required for glass materials to render properly.
+- SPPM Photon Target:
+    This flag indicates that the object should be a 'target' for the photons emitted from lamps in SPPM render mode.  Targeting a glass object this way can reduce rendering noise.
```
Comment (Mango-3) [inline docs/panels/object/object_options.rst]: This flag indicates that the object should be a 'target' for the photons emitted from lamps or the environment in SPPM render mode.  Targeting objects producing strong caustics (e.g. glass objects) will lead to significant improvements in render noise.

File: docs/features.rst [hunk 6]
```diff
@@ -1,19 +1,18 @@
...
-    * Full OSL materials
-    * Area lamps
-    * Support for linked objects and groups
-    * Archive assemblies
-    * Post processing stages
-    * The appleseed renderer is now bundled directly with blenderseed (no more configuration woes)
-    * Adjustable number of motion segments for camera, object, and deformation blur
+    * Blender 2.8 support
+    * Dynamic OSL script node
+    * Texture converter uses internal function (no more need for maketx)
```
Comment (Mango-3) [inline docs/features.rst]: More user friendly texture converter.

**Comment by jdent02** [comment]
Thanks!

**Pull request merged**